source/agora/consensus/protocol/Nominator.d

```diff
@@ -136,6 +136,9 @@ public extern (C++) class Nominator : SCPDriver
     public extern (D) void delegate (in ConsensusData data, in string msg)
         @safe onInvalidNomination;
 
+    /// Delegate called when a block is to be externalized
+    public extern (D) bool delegate (const ref Block) @safe externalizeBlock;
```

Comment:
Isn't the name of `acceptBlock` more proper because the `acceptBlock` of `FullNode` or `Validator` is called in the middle of externalizing the value?

Reply:
Yeah I guess it would be less confusing